src/App.jsx

```diff
@@ -18,17 +16,14 @@ export default function App() {
   }
 
   //DELETE
-  function handleDeleteTheme(idToRemove) {
- 
-    const filteredThemes = themes.filter(
-      (theme) => idToRemove !== theme.id
-    );
+  function handleOnDelete(idToRemove) {
```

Comment:
a typical naming convention is `handleDelelteTheme` (onDelete is for the prop name of the component).